ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -314,6 +314,27 @@
                     current-parameters: true
                   - name: 'ceph-ansible-prs-dev-centos-container-add_mdss'
                     current-parameters: true
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            #!/bin/bash
+            set -x
+            # if the target branch is master we RUN these tests.
+            if [[ "$ghprbTargetBranch" =~ "stable-" ]]; then
+              exit 1
+            fi
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-mon|tests/functional/add-rbdmirrors'
+          on-evaluation-failure: dont-run
+          steps:
+            - multijob:
+                name: 'ceph-ansible add_rbdmirrors playbook testing'
+                condition: SUCCESSFUL
+                execution-type: PARALLEL
+                projects:
+                  - name: 'ceph-ansible-prs-dev-centos-non_container-add_rbdmirrors'
+                    current-parameters: true
+                  - name: 'ceph-ansible-prs-dev-centos-container-add_rbdmirrors'
+                    current-parameters: true
```

Comment:
@rishabh-d-dave add the corresponding code for stable-4.0, I think it will be backported

Reply:
Done.